keyboards/sdrakbs/sdrakb00/keyboard.json

```
@@ -0,0 +1,79 @@
+{
+    "manufacturer": "Diego Andres Rabaioli",
+    "keyboard_name": "sdrakb00",
+    "maintainer": "drabaioli",
+    "bootloader": "caterina",
```

According to the linked [repository](https://github.com/drabaioli/SdraKb00), this uses an integrated ATmega32U4 microcontroller, which will use an `atmel-dfu` bootloader. Caterina bootloader only comes on Pro Micro development boards.
```suggestion
    "bootloader": "atmel-dfu",
```

---

If I implement this change then the firmware won't flash unless I specify `qmk ... -bl avrdude` as a cmd line parameter.

After soldering the PCB, no bootloader was available on the MCU, so I flashed `caterina` through the ISP pin.

---

Sounds like you may have bought the ATmega32U4RC, which comes completely blank and configured to use the internal 8MHz oscillator (which is not supported) rather than external crystal. Caterina is not an ideal choice; you should change it to the factory bootloader `atmel-dfu` as suggested.